src/building-blocks/FormField.tsx

```diff
@@ -56,22 +60,27 @@ class FormField extends Component<IProps> {
 
   public render () {
     const { form, fieldConfig } = this.props
-      , { getFieldDecorator } = form;
+      , { colProps, formItemProps, field, skipFieldDecorator } = fieldConfig
+      , { getFieldDecorator } = form
+      ;
 
     if (filterInsertIf(fieldConfig, form.getFieldsValue())) {
       return null;
     }
 
-    const FormItemComponent = (
-      <Antd.Form.Item {...fieldConfig.formItemProps} label={this.label}>
-        {getFieldDecorator(fieldConfig.field, this.decoratorOptions)(
-          <fieldConfig.editComponent {...this.editProps} />,
-        )}
-      </Antd.Form.Item>
+    const decoratorOptionsProp = skipFieldDecorator ? { decoratorOptions: this.decoratorOptions } : {}
```

Comment:
That whole block is quite complex ... too much ? I don't know.
I'm not sure how I'd clean it up either, so maybe it's fine as is.